src/server/components/dashboard/listsItems/item/update/actionHandlers/update.ts

```diff
@@ -0,0 +1,23 @@
+import { Request, Response } from "express";
+import { handleListItemUpdate } from "./publish";
+import { sendPublishedEmail } from "./helpers";
+import { ListItemWithAddressCountry } from "server/models/listItem/providers/types";
+
+export async function update(req: Request, res: Response): Promise<void> {
+  const userId = req.user!.id;
+  const listItem = res.locals.listItem;
+  const { listItemUrl, listIndexUrl } = res.locals;
+
+  try {
+    const [updatedListItem] = await handleListItemUpdate(listItem.id, userId);
```

Comment:
I've changed this function to return the result from prisma, so I can use the updated values directly instead querying again